### PR TITLE
track github interactions

### PR DESCRIPTION
**Pull request by christianalfoni**

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/4443/commits/e08293aa476f162b5fb8b378a3e6c6801154fabd is at https://pr4443.build.csb.dev/s/new.

File: packages/app/src/app/overmind/namespaces/git/actions.ts [hunk 1]
```diff
@@ -111,6 +111,7 @@ export const createRepoClicked: AsyncAction = async ({
   effects,
   actions,
 }) => {
+  effects.analytics.track('Create repo');
```
Comment (CompuIves) [inline packages/app/src/app/overmind/namespaces/git/actions.ts:114]:
```suggestion
  effects.analytics.track('GitHub - Create Repo');
```

File: packages/app/src/app/overmind/namespaces/git/actions.ts [hunk 2]
```diff
@@ -165,6 +166,7 @@ export const createRepoClicked: AsyncAction = async ({
 };
 
 export const openSourceSandbox: Action = ({ state, effects }) => {
+  effects.analytics.track('Open source sandbox');
```
Comment (CompuIves) [inline packages/app/src/app/overmind/namespaces/git/actions.ts:169]:
```suggestion
  effects.analytics.track('GitHub - Open Source Sandbox');
```

File: packages/app/src/app/overmind/namespaces/git/actions.ts [hunk 3]
```diff
@@ -177,6 +179,7 @@ export const createCommitClicked: AsyncAction = async ({
   effects,
   actions,
 }) => {
+  effects.analytics.track('Create commit');
```
Comment (CompuIves) [inline packages/app/src/app/overmind/namespaces/git/actions.ts:182]:
```suggestion
  effects.analytics.track('GitHub - Create Commit');
```

File: packages/app/src/app/overmind/namespaces/git/actions.ts [hunk 4]
```diff
@@ -249,6 +252,7 @@ export const createPrClicked: AsyncAction = async ({
   effects,
   actions,
 }) => {
+  effects.analytics.track('Open PR');
```
Comment (CompuIves) [inline packages/app/src/app/overmind/namespaces/git/actions.ts:255]:
```suggestion
  effects.analytics.track('GitHub - Open PR');
```

File: packages/app/src/app/overmind/namespaces/git/actions.ts [hunk 5]
```diff
@@ -337,9 +341,10 @@ export const updateGitChanges: Operator = pipe(
 );
 
 export const resolveConflicts: AsyncAction<Module> = async (
-  { state, actions },
+  { state, actions, effects },
   module
 ) => {
+  effects.analytics.track('Resolve conflicts');
```
Comment (CompuIves) [inline packages/app/src/app/overmind/namespaces/git/actions.ts:347]:
```suggestion
  effects.analytics.track('GitHub - Resolve Conflicts');
```

File: packages/app/src/app/overmind/namespaces/git/actions.ts [hunk 6]
```diff
@@ -423,6 +428,7 @@ export const resolveOutOfSync: AsyncAction = async ({
   actions,
   effects,
 }) => {
+  effects.analytics.track('Resolve out of sync');
```
Comment (CompuIves) [inline packages/app/src/app/overmind/namespaces/git/actions.ts:431]:
```suggestion
  effects.analytics.track('GitHub - Resolve out of sync');
```

**Pull request merged**